pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -129,14 +129,14 @@
  * TODO: migrate code in this class to other places for better readability
  */
 public class PinotLLCRealtimeSegmentManager {
+
+  // simple field in Ideal State representing pause status for the table
+  public static final String IS_TABLE_PAUSED = "isTablePaused";
```

Comment:
instead of exposing a private static field. we can probably expose the isTablePause method as public static. but can be follow up